lib/ain-rs-exports/src/lib.rs

```diff
@@ -98,6 +99,9 @@ pub mod ffi {
         ) -> Vec<u8>;
 
         fn evm_disconnect_latest_block();
+
+        fn evm_get_block_hash_by_number(height: u64) -> [u8; 32];
```

Comment:
Please use `CrossBoundaryResult` so any actual errors from inner APIs aren't swallowed and instead propagated to layers up, and differentiate between 0 and a real error.

Reply:
Fixed in https://github.com/DeFiCh/ain/pull/2162/commits/edc4ea7ef3f3206e67a8c0c7c04a535cc42df68f